os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.h

```diff
@@ -77,59 +77,66 @@ extern VOID RtlUdelayOS(u32 us);
 // other MCU may use standard library 
 #include <string.h>
 #endif
+
+
+#if (defined CONFIG_GSPI_HCI || defined CONFIG_SDIO_HCI) || defined(CONFIG_LX_HCI)
+/* For SPI interface transfer and us delay implementation */
+#if !defined(CONFIG_PLATFORM_8195A) && !defined(CONFIG_PLATFORM_8711B) && !defined(CONFIG_PLATFORM_8721D)  && !defined(CONFIG_PLATFORM_8195BHP) && !defined(CONFIG_PLATFORM_8710C)
+#include <rtwlan_bsp.h>	
+#endif
+#endif
+
+
+// --------------------------------------------
+//	Platform dependent type define
+// --------------------------------------------
 #ifndef FIELD_OFFSET
-#define FIELD_OFFSET(s, field) ((SSIZE_T) & ((s *)(0))->field)
+#define FIELD_OFFSET(s,field)	((SSIZE_T)&((s*)(0))->field)
 #endif
 
 // os types
-typedef char osdepCHAR;
-typedef float osdepFLOAT;
-typedef double osdepDOUBLE;
-typedef long osdepLONG;
-typedef short osdepSHORT;
-typedef unsigned long osdepSTACK_TYPE;
-typedef long osdepBASE_TYPE;
-typedef unsigned long osdepTickType;
-
-// lock and semaphore
-typedef void*	            _timerHandle;
-typedef void *_sema;
-typedef void *_mutex;
+typedef char			    osdepCHAR;
+typedef float			    osdepFLOAT;
+typedef double			    osdepDOUBLE;
+typedef long			    osdepLONG;
+typedef short			    osdepSHORT;
+typedef unsigned long	    osdepSTACK_TYPE;
+typedef long			    osdepBASE_TYPE;
+typedef unsigned long	    osdepTickType;
 
+typedef void*	            _timerHandle;
+typedef void*	            _sema;
+typedef void*	            _mutex;
 typedef void*	            _lock;
+typedef void*	            _queueHandle;
+typedef void*	            _xqueue;
+typedef struct timer_list	_timer;
 
-typedef void *_queueHandle;
-typedef void *_xqueue;
-typedef struct timer_list _timer;
-
-typedef int (*task_fun)(int argc, char *argv[]);
-
-#define _SUCCESS 1
-#define _FAIL 0
-#define VOID void
+#define rtw_timer_list timer_list
 
-typedef struct sk_buff _pkt;
-typedef unsigned char _buffer;
+typedef	struct sk_buff		_pkt;
+typedef unsigned char		_buffer;
+typedef unsigned int        systime;
 
-#ifndef __LIST_H
+#if !defined(__LIST_H) && !defined(__LIST_H__) 
 #warning "DLIST_NOT_DEFINE!!!!!!"
 struct list_head {
 	struct list_head *next, *prev;
 };
 #endif
 
-struct __queue {
-	struct list_head queue;
-	_lock lock;
+struct	__queue	{
```

Comment:
Is this the correct tab?

Reply:
Hey I have double checked this, it seems to be the right tab. Should a tab be 1 space or 4 spaces?

Reply:
@jsdosa any update on this?

Reply:
@jsdosa any update on this?